manifest/signature_test.go

```diff
@@ -56,7 +63,10 @@ func Test_HashModule(t *testing.T) {
 	}
 
 	graph, _ := NewModuleGraph(manifest.Modules)
-	hashes := NewModuleHashes()
+	params := map[string]string{
+		"map_pools_created": "foo",
+	}
+	hashes := NewModuleHashes(WithModuleHashesParams(params))
```

Comment:
ok I think I understand the confusion. The `Value` in `pbsubstreams.Module_Input_Params` is truly the _actual_ param.. not the type, but the contents of the params. So we don't need to pass them out of band.